docs/src/content/hardhat-runner/docs/config/index.md

```diff
@@ -208,3 +208,20 @@ You can configure how your tests are run using the `mocha` entry, which accepts
 ## Quickly integrating other tools from Hardhat's config
 
 Hardhat's config file will always run before any task, so you can use it to integrate with other tools, like importing `@babel/register`.
+
+## Default EVM Version
+
+The default EVM version is determined by solc's choice for a given compiler version. To specify a different EVM version, modify your `hardhat.config.js`:
+
+```js
+module.exports = {
```

Comment:
Just an idea: I saw that in one of the tests you do a "more complex override":

```
{
        solidity: {
          compilers: [
            { version: "0.8.20", settings: { evmVersion: "istanbul" } },
          ],
          overrides: {
            "contracts/ERC20.sol": {
              version: "0.8.21",
              settings: { evmVersion: "shanghai" },
            },
          },
        },
      }
```

Could it be useful for the users to also have a similar example in the docs?

Reply:
I believe that's already covered in the [multiple solidity versions](https://hardhat.org/hardhat-runner/docs/advanced/multiple-solidity-versions) guide, and adding it here might be a bit distracting. Using overrides is an advanced use case, and changing the evm version is also an advanced use case, so the combination of both things doesn't seem worth being explicitly mentioned.